### DISCUSSION TITLE
Activity and Notifications

### DISCUSSION THREAD

**Discussion by gabestein** [knowledgefutures/pubpub#1382] [Ideas]
We'll soon be moving into building a new system on PubPub: activity and notifications!

In a slight change to how we've approached building other systems, we came in with a very strong opinion: we believe that PubPub should not be a "sticky" platform. That is, we don't want to build features that, intentionally or not, create the kind of habit-forming loops that we see in notifications from social media and other types of platforms. At the same time, of course, we want to give Communities the tools they need to understand what's happening in their Communities and respond in appropriate and timely ways. Thus, rather than run a specific design process with Communities like we're doing with editorial management, we've spent the last few months slowly collecting requirements from a number of folks and merging them with our initial strongly held opinions.

What follows is a short summary of what we're planning to build out in the coming weeks. As always, we welcome your thoughts and feedback — particularly if you're wondering about solving for a specific use-case.

## Activity List View
The main view that shows activity to Community Members.

- Clickable list of activities at every scope
- Ability to choose to receive email digest and frequency
- Ability to filter list by types of activities

## Member Email Digest
An opt-in digest sent to Community Members

- Digest that contains activity within the community since the last digest was sent
- Email template that uses some community logos/colors
- Setting on activity page to choose whether to receive digest, and frequency

## User Invites
Emails sent to users when they are added to Communities/Pubs/Collections

- Email to existing users when they are added as members at any scope, linking to overview dashboard at that scope
- Ability to add new users who are not already PubPub users, via email

## Subscription (non-member) Emails
Emails sent to users who are not Members of the Community. We're starting very slowly here, with the idea of expanding the feature over time to incorporate new ideas of what it means to opt in to being part of a community (vs. being explicitly added by an admin, as suggested by @oliviodare https://github.com/pubpub/pubpub/discussions/1323#discussioncomment-533773).

- New replies to a discussion

**Comment by jeffpooley** [comment]
Everything outlined here is extremely exciting, including the explicit avoidance of “sticky” design.
